qiskit_experiments/library/hamiltonian/heat_base.py

```diff
@@ -0,0 +1,243 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+Base Class for general Hamiltonian Error Amplifying Tomography experiments.
+"""
+
+from abc import ABC
+from typing import List, Tuple, Optional
+
+from qiskit import circuit, QuantumCircuit
+from qiskit.providers import Backend
+
+from qiskit_experiments.framework import BaseExperiment, BatchExperiment, Options
+from qiskit_experiments.curve_analysis import ParameterRepr
+from .heat_analysis import HeatElementAnalysis, HeatAnalysis
+
+
+class HeatElement(BaseExperiment):
+    """Base class of HEAT experiment elements.
```

Comment:
This is not HEAT experiment by itself. This provides ping-pong that amplifies composite error terms and HEAT combines them to resolve.

Reply:
Maybe that point should be made earlier on in the docstring here.

Reply:
Also, maybe some of the explanation of how heat works should be pulled from here into a generic HEAT overview section (in `hamiltonian/__init__.py` or `mod_hamiltonian.rst`?), so that this docstring could just describe where this class fits into the hierarchy of the different classes? Also, perhaps it should reference ZXHeat as a complete working example?